lib/yao/resources.rb

```diff
@@ -20,6 +20,7 @@ module Resources
 
     autoload :Resource,          "yao/resources/resource"
     autoload :Meter,             "yao/resources/meter"
+    autoload :OldSample,         "yao/resources/old_sample"
     autoload :Sample,            "yao/resources/sample"
```

Comment:
Oh I found `Sample` is different from `OldSample`

http://docs.openstack.org/developer/ceilometer/webapi/v2.html#Sample